Sources/XCTest/Public/XCTestSuite.swift

```diff
@@ -38,6 +38,27 @@ open class XCTestSuite: XCTest {
         return XCTestSuiteRun.self
     }
 
+    #if USE_SWIFT_CONCURRENCY_WAITER
+    override func _performAsync(_ run: XCTestRun) async {
+        guard let testRun = run as? XCTestSuiteRun else {
+            fatalError("Wrong XCTestRun class.")
+        }
+
+        run.start()
+        func syncSetUp() { setUp() }
```

Comment:
What's the motivation for wrapping the `setUp` call in a helper function like this (and similarly below for tearDown)? Does it influence the executor that the method runs on?

Reply:
Ohhhhh yes, of course. Thanks for that reminder.